.gitignore

```diff
@@ -6,6 +6,7 @@
 *$
 *.bak
 *.diff
+.idea
```

Comment:
Sure, I can add a `/`. PyCharm doesn't seem to generate the other ones, but I can add them. I think there is a Python plugin for IntelliJ.